selene-lib/default_std/roblox_base.yml

```
@@ -262,6 +262,14 @@ globals:
     args:
       - type: "..."
     must_use: true
+  Font.new:
+    args:
+      - type: string
+      - required: false
+        type: Enum.FontWeight
+      - required: false
+        type: Enum.FontStyle
```

Currently erroring, try display types (referenced [here](https://kampfkarren.github.io/selene/usage/std.html#argument-types))

```suggestion
        type:
          display: "Enum.FontWeight"
      - required: false
        type:
          display: "Enum.FontStyle"
```